### PR TITLE
Wiki Update <Development>: Added a section for migration ideas

### PR DESCRIPTION
**Pull request by petermarshall878**
### Outline

Added a new section under Development called Migration, under which we can keep track of ideas / solutions from Prime that may carry over to RA. First entry under Migration is to discuss browser caching as implemented in Prime and now present in RA. 

Reference: Architecture Issue #67

###### Check List

- [x] commits are squashed 
- [x] commits follow the [Karma format][karma-format]

[karma-format]: karma-runner.github.io/1.0/dev/git-commit-msg.html

### CODE REVIEW

**Comment by ahmadnassri** [comment]
I'd rather keep this wiki free of "Prime" references and focus on an abstract architectural patterns. especially since this is a public wiki!

I would rephrase and re-structure the doc in context of caching best practices only

File: migration/caching.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+The bottom-line is that the user should not see their private information after logging out should they click on the “Back” button of their browser. A solution is live in Prime (as of early July, 2017) and must be considered in RA. It is currently present in parts of RA (see team Nutella or details below).
+
+The negative side-effects are currently being monitored in Prime (particularly performance). 
+
+
+## How
+
+The solution that is currently in production for Prime is to disable caching by adding “no-cache” and “no-store” directives in the headers from within our Prime_Controller.php. The 5 lines of code added at the head of Prime_Controller.php are:
+
+header("Cache-Control: no-store, no-cache, must-revalidate");
```
Comment (darcyclarke) [inline migration/caching.md]: I think this should probably be in a code block for better legibility. You can start and end one with <code>```</code> 

ex. 

> <code>```js</code> 
>
> [your code here]
>
> <code>```</code>

**Comment by ahmadnassri** [comment]
@petermarshall878 can you rebase off master plz?

**Comment by petermarshall878** [comment]
Squashed commits and fork has been merged with master. Thank you!
